Userland/Libraries/LibJS/Token.h

```diff
@@ -181,7 +181,7 @@ class Token {
 public:
     Token() = default;
 
-    Token(TokenType type, DeprecatedString message, StringView trivia, StringView value, StringView filename, size_t line_number, size_t line_column, size_t offset)
+    Token(TokenType type, String const& message, StringView trivia, StringView value, StringView filename, size_t line_number, size_t line_column, size_t offset)
```

Comment:
This should still take the string by value (and then `move()` it into `m_message`, which is currently missing)

Reply:
Fixed